coldfront/core/utils/templates/mou_template.html

```diff
@@ -0,0 +1,65 @@
+<html>
+<head>
+<style>
+* {
+  font-family: 'Bitstream Charter';
+  font-size: 20px;
+}
+
+table {
+  float: right;
+  border-collapse: collapse;
+}
+
+td {
+  border: 1px solid black;
+  padding: 5px;
+}
+
+.gray {
+  color: gray;
+  font-size: large;
+}
+
+.mou {
+  color: gray;
+  font-size: 40px;
+  font-weight: bold;
+  text-align: right;
+}
+
+.signature {
+  float: left;
+}
+
+</style>
+</head>
+<body>
+<img src="/vagrant/coldfront_app/coldfront/coldfront/static/core/portal/imgs/brc_logo.png" alt="BRC Logo" style="float: left;" height=200>
```

Comment:
This file path isn't going to work in production.

Reply:
"{{ static_root }}/core/portal/imgs/brc_logo.png" works. Note we can't use {% static 'core/portal/imgs/brc_logo.png' %} because the pdf conversion requires local paths and not paths on the website